src/monodroid/monodroid.targets

```diff
@@ -13,7 +13,7 @@
       Outputs="@(_MonoRuntime->'$(OutputPath)\%(Identity)\libmono-android.$(_Conf).so')">
     <Which Program="%(_RequiredProgram.Identity)" Required="True" />
     <PropertyGroup>
-      <_AppAbi>$(AndroidSupportedAbis.Replace(':', ' '))</_AppAbi>
+      <_AppAbi>@(_MonoRuntime->'%(Identity)', ' ')</_AppAbi>
```

Comment:
The intent is to create a ` ` (space)-separated list of ABI names. `@(_MonoRuntime)` could be e.g. `armeabi-v7a;x86`, but `$(AppApi)` would need to be `armeabi-v7a x86` (`s/;/ /g`).